editoast/src/converters/osm_to_railjson.rs

```diff
@@ -148,12 +144,14 @@ mod tests {
 
     #[test]
     fn parse_switches() {
+        /// Need changes because of links turning into switches
```

Comment:
```suggestion
        /// TODO: Need changes because of links turning into switches
```

Reply:
I'll remove this comment, I already adapt tests.